### ISSUE TITLE
Improving Intelligence report

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#668]
Currently, the Intel report is 60% accurate at Level 1, 50% at Level 2, and 40% at Level 3.  However, it seems like, if you watch a prestige first of something like Three-Person Capsule, the CIA _should_ add Soyuz to the Statistics page in Intel, with a (possibly) wild guess as to its reliability.

**Comment by hweimer** [comment]
Good idea, but that should be done for any piece of hardware that is visible in a video. However, it's quite tricky to map video filenames back to hardware components, so you should probably cheat a bit and look into the mission hardware table to see what actually has been used.
